UI.tsv

```diff
@@ -456,8 +456,8 @@ UI_20150317_000455	{@st41}Quest Sharing
 UI_20150317_000456	{@st41b}Enabled
 UI_20150317_000457	{@st41b}Disabled
 UI_20150317_000458	{@st41b}Create Party
-UI_20150317_000459	{@st43} Public Party List {/}
-UI_20150317_000460	{@st59b} Public Party List {nl} {@st59s} - Search for parties you want to join {nl} - Use search conditions to search the party you want {/}
+UI_20150317_000459	{@st43}Public Party List {/}
+UI_20150317_000460	${@st59b}Public Party List{nl}{@st59s} - Search for parties you want to join. {nl} - Tick the "Use Search" box to search for specific conditions to join your desired party. {/}
```

Comment:
The last sentence doesn't sound right.
Maybe something like: "Ticking "Use Search" uses your search conditions to help find desirable parties"